centreon/src/Core/HostTemplate/Infrastructure/Repository/DbReadHostTemplateRepository.php

```diff
@@ -103,23 +103,19 @@ public function findByRequestParameter(RequestParametersInterface $requestParame
                     ehi.ehi_icon_image_alt,
                     hcr.hostcategories_hc_id as severity_id
                 FROM `:db`.host h
-                LEFT JOIN `:db`.extended_host_information ehi ON h.host_id = ehi.host_host_id
-                LEFT JOIN `:db`.hostcategories_relation hcr ON h.host_id = hcr.host_host_id
+                LEFT JOIN `:db`.extended_host_information ehi
+                    ON h.host_id = ehi.host_host_id
+                LEFT JOIN `:db`.hostcategories_relation hcr
+                    ON hcr.host_host_id = h.host_id
+                LEFT JOIN `:db`.hostcategories hc
```

Comment:
The where clause disapear but here we have a `LEFT JOIN` instead of `INNER JOIN`.
I may have not understood, but it seems weird to me at first glance.
```suggestion
                INNER JOIN `:db`.hostcategories hc
```

Reply:
Discussed, it's a feature. ✅